CONTRIBUTING.md

```diff
@@ -25,6 +25,7 @@ Head over to our public [repository](https://github.com/arnaudmiribel/streamlit-
     __author__ = "Eva Jensen"
     __experimental_playground__ = False # Optional
     ```
+    To test it out, run `poetry run` and `pip install -e .` in the `streamlit-extras` directory, and then run the `gallery/streamlit_app.py` app.
```

Comment:
`pip install -e .` alone should be sufficient

Reply:
great, removed poetry run!